src/Pact/Types/Hash.hs

```diff
@@ -39,6 +39,7 @@ import Data.Text.Encoding (decodeUtf8, encodeUtf8)
 import Data.Reflection
 import Data.Proxy
 import Test.QuickCheck
+import Test.QuickCheck.Instances()
```

Comment:
?

Reply:
Necessary for arbitrary instances.

Our `SizeOf` type was propagating `Arbitrary` before which didn't really make sense, only because of this module.

I removed the need for `Arbitrary`, but now we need this import for `Hash` to compile.